apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -2322,19 +2245,11 @@ static void addCommonEobInformationInpatientSNF(
     }
 
     // diagnosisRelatedGroupCd
-    // CLM_DRG_CD => ExplanationOfBenefit.diagnosis
-    if (diagnosisRelatedGroupCd.isPresent()) {
-      /*
-       * FIXME This is an invalid DiagnosisComponent, since it's missing a (required)
-       * ICD code. Instead, stick the DRG on the claim's primary/first diagnosis.
-       * SamhsaMatcher uses this field so if this is updated you'll need to update
-       * that as well.
-       */
-      eob.addDiagnosis()
-          .setSequence(1)
-          .setPackageCode(
-              createCodeableConcept(eob, CcwCodebookVariable.CLM_DRG_CD, diagnosisRelatedGroupCd));
-    }
+    // CLM_DRG_CD => ExplanationOfBenefit.supportingInfo
```

Comment:
This was incorrectly being mapped into a DiagnosisComponent (this was carried forward from STU3) and has been updated to map to the field dictated in the Profile mapping.